src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
+      <div class="calendar_day"></div>
```

Comment:
It seems like you're creating a static calendar with 31 days always starting on Sunday. But what if we need to represent a different month, or start the week from a different day? Consider making your calendar more dynamic. You can achieve this by using JavaScript to generate the calendar days based on the month and the start day.